src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerEnclaveProvider.java

```diff
@@ -148,7 +161,7 @@ default void processAev1SPDE(String userSql, String preparedTypeDefinitions, Par
 
         // Process the second resultset.
         if (!stmt.getMoreResults()) {
-            throw new SQLServerException(this, SQLServerException.getErrString("R_UnexpectedDescribeParamFormat"), null,
+            throw new SQLServerException(null, SQLServerException.getErrString("R_UnexpectedDescribeParamFormat"), null,

```

Comment:
why is this null

Reply:
It's an AEv1 thing msg that we don't really even handle in AEv2. It needed to be null because I considered putting it in Util file. Still not sure where the best place is for these methods, looking to see if I can refractor AEv1 too.